orca-queue/src/main/kotlin/com/netflix/spinnaker/orca/q/handler/StartWaitingExecutionsHandler.kt

```diff
@@ -51,11 +51,10 @@ class StartWaitingExecutionsHandler(
             ?.let { newest ->
               log.info("Starting queued {} {} {}", newest.application, newest.name, newest.id)
               queue.push(StartExecution(newest))
-              (pipelines - newest)
-                .also { queued ->
-                  log.info("Dropping queued {} {} {}", newest.application, newest.name, queued.map { it.id })
```

Comment:
The fact that this logging statement is complete fucking nonsense didn't help me debug things.